juanevillam/xtrovert
# Improve Auth Layout for Readability and Responsiveness

## Overview  
This PR enhances the Auth layout components by improving readability and responsiveness. It utilizes better HTML attributes, reduces unnecessary HTML tags, and enhances accessibility features.

---

## Key Changes  

### **Enhancements**  
- Replaced `<strong>` tags with the `font-semibold` class in the `AuthLink` component.  
- Improved accessibility and user experience for the `Button` component.  
- Added `AlertTriangleImage` for better error indication in `AuthTextInput` and improved the component's structure.  

---

## Reasons for Changes  
These changes improve the overall user experience by enhancing code readability, increasing accessibility, and ensuring better error indications. This leads to a more maintainable codebase and a more user-friendly interface.

---

## How to Test  
1. Run the application and navigate to the authentication pages.  
2. Verify that the Auth layout displays correctly and is responsive on different devices.  
3. Test the `AuthLink`, `Button`, and `AuthTextInput` components to ensure they function properly.  
4. Check that error indications appear correctly when invalid input is provided.  
5. Ensure that accessibility features (like proper HTML attributes) are correctly implemented.
